pkg/dispatchcli/cmd/create.go

```diff
@@ -67,7 +69,18 @@ func importFile(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []stri
 			return errors.Wrapf(err, "Error decoding document %s", string(doc))
 		}
 		switch docKind := k.Kind; docKind {
-		case "base-image":
+		case "API":
```

Comment:
String value `"API"` is the same as `APIKind` const above. 

Would you consider moving the `*Kind` constants to a package which can be imported everywhere they are needed, so constants could be used instead of string values?

Reply:
I think this is a great idea. Do you have a recommendation of where I should create this constants package?